platforms/gke/base/use-cases/inference-ref-arch/kubernetes-manifests/online-inference-gpu-h100-llama4-scout/online-inference-gpu-deployment.yaml

```diff
@@ -0,0 +1,213 @@
+                  name: model-download-configmap
+          volumeMounts:
+            - name: gcsfuse
+              mountPath: /gcs
+              readOnly: true
+        - name: inference-server
+          args:
+            - --disable-log-requests
+            - --gpu-memory-utilization=0.95
+            - "--model=/gcs/$(MODEL_ID)"
+            - --tensor-parallel-size=8
+            - --trust-remote-code
+            - --max-model-len=1000000
+          env:
+            - name: MODEL_ID
+              valueFrom:
+                configMapKeyRef:
+                  key: MODEL_ID
+                  name: model-download-configmap
+          image: vllm/vllm-openai:v0.8.4
+          imagePullPolicy: Always
+          readinessProbe:
+            failureThreshold: 6000
+            httpGet:
+              path: /health
+              port: 8000
+              scheme: HTTP
+            initialDelaySeconds: 60
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 1
+          ports:
+            - containerPort: 8000
+              name: metrics
+          resources:
+            requests:
+              cpu: "64"
+              memory: "512G"
+              nvidia.com/gpu: "8"
+            limits:
+              cpu: "64"
+              memory: "512G"
+              nvidia.com/gpu: "8"
+          volumeMounts:
+            - mountPath: /dev/shm
+              name: dshm
+            - name: gcsfuse
+              mountPath: /gcs
+              readOnly: true
+      nodeSelector:
+        cloud.google.com/gke-accelerator: nvidia-h100-80gb
+      serviceAccountName: ira-online-gpu-ksa
+      tolerations:
+        - key: "nvidia.com/gpu"
+          operator: "Exists"
+          effect: "NoSchedule"
+        - key: "on-demand"
+          value: "true"
+          operator: "Equal"
+          effect: "NoSchedule"
+      volumes:
+        - name: dshm
+          emptyDir:
+            medium: Memory
+        - name: gcsfuse
+          csi:
+            driver: gcsfuse.csi.storage.gke.io
+            volumeAttributes:
+              bucketName: cloud-storage-bucket-name
+              # Leave the only-dir option at the end of the mountOptions value
+              # because Kustomize will take care of replacing it
+              mountOptions: "metadata-cache:ttl-secs:-1,metadata-cache:stat-cache-max-size-mb:-1,metadata-cache:type-cache-max-size-mb:-1,metadata-cache:negative-ttl-secs:0,file-cache:max-size-mb:-1,file-cache:cache-file-for-range-read:true,file-cache:enable-parallel-downloads:true,implicit-dirs,file-system:kernel-list-cache-ttl-secs:-1,only-dir:MODEL_ID"
+              skipCSIBucketAccessCheck: "true"
+        - name: gke-gcsfuse-cache
+          emptyDir:
+            medium: Memory
+        - name: gke-gcsfuse-tmp
+          emptyDir:
+            medium: Memory
+        - name: gke-gcsfuse-buffer
+          emptyDir:
+            medium: Memory
+---
+apiVersion: monitoring.googleapis.com/v1
```

Comment:
As far as I know, this is being created automatically if you p[Configure automatic application monitoring for workloads](https://cloud.google.com/kubernetes-engine/docs/how-to/configure-automatic-application-monitoring)